docs/usage.md

```diff
@@ -141,9 +135,9 @@ If you don't want run eggNOG-mapper, you will need to add the flag `--skip_eggno
 
 ## Example pipeline command with some common features
 
-`nextflow run lnuc-eemis/metatdenovo -profile docker --input samplesheet.csv --assembler rnaspades --orf_caller transdecoder --eggnog --run_dbcan`
+`nextflow run lnuc-eemis/metatdenovo -profile docker --input samplesheet.csv --assembler rnaspades --orf_caller transdecoder --eggnog`
```

Comment:
```suggestion
```nextflow
nextflow run lnuc-eemis/metatdenovo -profile docker --input samplesheet.csv --assembler rnaspades --orf_caller transdecoder --eggnog
```
```
By having it as a code block with syntax highlighting you get also a copy-button rendered along with it on the website